src/ntc-temperature-sensor-element.ts

```diff
@@ -111,15 +111,13 @@ export class NTCTemperatureSensorElement extends LitElement {
           stroke-linejoin="miter"
           stroke-width=".65px"
         />
-        <g fill="#fffefe" font-family="sans-serif">
```

Comment:
Great! Now just reformat the file with prettier to keep the indentation correct ( F1 then "Format Document" inside gitpod / vscode should probably do the trick)

Reply:
yes, thanks!